akka-actor/src/main/scala/akka/event/Logging.scala

```diff
@@ -877,15 +885,6 @@ object Logging {
 
     import StdOutLogger._
 
-    // format: OFF
-    // FIXME: remove those when we have the chance to break binary compatibility
```

Comment:
I guess this comment wasn't valid as they are private?

Reply:
@chbatey I hesitated in removing those but the four Vals were just _not_ used by anything and eventually thought it was time to remove.